pkg/metrics/metrics.go

```diff
@@ -33,19 +33,21 @@ var (
 			Help:      "Gauge of all sandbox namespaces that are active.",
 		})
 
-	specsLoaded = prometheus.NewGaugeVec(
+	specs = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Subsystem: subsystem,
-			Name:      "specs_loaded",
-			Help:      "Specs loaded from registries, partitioned by registry name.",
-		}, []string{"registry_name"})
-
-	specsReset = prometheus.NewCounter(
-		prometheus.CounterOpts{
-			Subsystem: subsystem,
-			Name:      "specs_reset",
-			Help:      "Counter of how many times the specs have been reset.",
-		})
+			Name:      "source",
```

Comment:
I think the name should be something like `specs_total`. https://prometheus.io/docs/practices/naming/

Reply:
Good catch @shawn-hurley, I did not intend it to be this way. Also, I think, we are staying consistent in giving variable name and CounterOpts Name param, I am renaming variable `specs` to `specsTotal` (along with the change you suggested ofcourse)